### DISCUSSION TITLE
Raspberry Pi 4

### DISCUSSION THREAD

**Discussion by WillChi69** [frntc/Sidekick64#46] [Q&A] [answered]
I would like to ask whether somebody was useful already Raspberry Pi 4? Work with properly?
Thank You for information!

**Comment by frntc** [comment, marked as answer]
The firmware has been compiled and there have been some tests, but no real efforts to make it run on a RPi4. The RPi4 behaves differently than the SoC on the RPi3A+/B+/Zero2 and this needs some more investigation (no promises when/if this will happen!).

**Reply by WillChi69** [reply]
Thank You for this information!
